tool/getsubdomains.go

```diff
@@ -0,0 +1,73 @@
+package tool
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/fatih/color"
+)
+
+type Response []struct {
```

Comment:
It would be better to define a response item instead of declaring the response as an array IMHO